deepchecks/vision/checks/model_evaluation/weak_segments_performance.py

```diff
@@ -60,67 +56,52 @@ class WeakSegmentsPerformance(SingleDatasetCheck, WeakSegmentAbstract):
         - ``'categorical'`` - for discrete, non-ordinal outputs. These can still be numbers,
           but these numbers do not have inherent value.
 
-        For more on image properties, see the guide about :ref:`vision_properties_guide`.
-    number_of_bins: int, default : 5
-        Maximum number of bins to segment a single property into.
-    number_of_samples_to_infer_bins : int, default : 1000
-        Minimum number of samples to use to infer the bounds of the segments' bins
-    n_top_properties: int , default: 10
-        Number of features to use for segment search. Top columns are selected based on feature importance.
-    n_to_show: int , default: 3
-        number of segments with the weakest performance to show.
+        For more on image properties, see the guide about :ref:`vision__properties_guide`.
     segment_minimum_size_ratio: float , default: 0.05
         Minimum size ratio for segments. Will only search for segments of
         size >= segment_minimum_size_ratio * data_size.
+    n_samples : Optional[int] , default: 10_000
+        number of samples to use for this check.
+    n_to_show : int , default: 3
+        number of segments with the weakest performance to show.
+    categorical_aggregation_threshold : float , default: 0.05
```

Comment:
```suggestion
    category_frequencey_threshold : float , default: 0.05
```

IMO better but not critical